logprep/processor/domain_resolver/processor.py

```diff
@@ -50,6 +51,13 @@
 logger = logging.getLogger("DomainResolver")
 
 
+class ResolveStatus(Enum):
```

Comment:
Consider using an `IntEnum` here as you use `ResolveStatus.SUCCESS` in return of methods, which evaluates now to a EnumType. By using `IntEnum` it resolves to an primitive integer, which should use less memory I guess. 
Please provide a docstring for this class.